.travis.yml

```diff
@@ -0,0 +1,90 @@
+conditions: v1
+
+git:
+  depth: false
+
+language: python
+
+dist: trusty
+cache: pip
+
+before_install:
+- &install-flit >-
+  pip install --upgrade pip flit
+
+.mixtures:
+- &run-if-tagged
+  if: tag IS present
+- &run-if-cherry-picker
+  if: tag =~ ^cherry\-picker\-v\d+\.\d+\.\d+$
```

Comment:
This can be hugely simplified now

Reply:
Can you suggest how? I'm not actually very well-versed in config file like this 😅

Reply:
Just use that other anchor without a regex